FALCON/Source/FalconMakeGlobalModel.m

```diff
@@ -236,12 +236,13 @@
     
 end
 
-Page1 = {};
-Page1 = [Page1; InNamesSheet(:, :, 1)];
-Page1 = [Page1; num2cell(InputsSheet(:, :, 1))];
-
-Page2 = {};
-Page3 = {};
+for m = 1:length(MeasFileList)
+    Page1 = {};
```

Comment:
Page1 should not be reinitialized within the loop
de facto this collects the names of the last file instead of the first
does not solve the problem